### PR TITLE
NETOBSERV-1111: Add support for RoundTripTime in FLP.

### PR DESCRIPTION
**Pull request by dushyantbehl**
Related PR - https://github.com/netobserv/netobserv-ebpf-agent/pull/117

Since the above pull request makes changes in the ebpf agent proto buf which it exports, this PR is marked as breaking changes as we need same version of agent and flp to work together.

### CODE REVIEW

**Review by eranra — DISMISSED** [review]

**Review by KalmanMeth — APPROVED** [review]

File: pkg/pipeline/decode/decode_protobuf.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ func PBFlowToMap(flow *pbflow.Record) config.GenericMap {
 		"Flags":           flow.Flags,
 		"IcmpType":        flow.Icmp.GetIcmpType(),
 		"IcmpCode":        flow.Icmp.GetIcmpCode(),
+		"RoundTripTimeMs": flow.TimeFlowRtt.AsDuration().Milliseconds(),
```
[the remark this answers is not in the document]
Comment (dushyantbehl) [reply pkg/pipeline/decode/decode_protobuf.go:60]: Sure..will update I think `TimeFlowRTTMs` will match your example is that fine @eranra

[the remark this answers is not in the document]
Comment (dushyantbehl) [reply pkg/pipeline/decode/decode_protobuf.go]: Updated - TimeFlowRttMs

**Comment by dushyantbehl** [comment]
@KalmanMeth @eranra  do we need to hold these changes till the PR:https://github.com/netobserv/netobserv-ebpf-agent/pull/117 gets merged?

**Review by jpinsonneau — DISMISSED** [review]
Looks good, thanks @dushyantbehl !

**Review by eranra — DISMISSED** [review]
Looks good to me :-)

**Comment by KalmanMeth** [comment]
> @KalmanMeth @eranra do we need to hold these changes till the PR:[netobserv/netobserv-ebpf-agent#117](https://github.com/netobserv/netobserv-ebpf-agent/pull/117) gets merged?

@dushyantbehl Yes, I think we need to first merge the ebpf PR and then the FLP PR.

**Comment by dushyantbehl** [comment]
Rebased. Please take a look @eranra @jpinsonneau @KalmanMeth

**Review by jpinsonneau — APPROVED** [review]
LGTM in terms of code, thanks @dushyantbehl !

**Comment by dushyantbehl** [comment]
Thanks @jpinsonneau!!

**Comment by dushyantbehl** [comment]
@jpinsonneau @eranra @KalmanMeth 
Can one of you please approve these changes as the PR https://github.com/netobserv/netobserv-ebpf-agent/pull/117 has been merged 😄

**Comment by jpinsonneau** [comment]
> @jpinsonneau @eranra @KalmanMeth Can one of you please approve these changes as the PR [netobserv/netobserv-ebpf-agent#117](https://github.com/netobserv/netobserv-ebpf-agent/pull/117) has been merged 😄

@dushyantbehl before approving this you need to update your ebpf agent dependency in go.mod

**Review by KalmanMeth — APPROVED** [review]

**Comment by jotak** [comment]
thanks @dushyantbehl !
/lgtm
/approve

**Pull request merged**